### PR TITLE
Fix diff docstrings.

### LINKED ISSUE #427: Use tf.convert_to_tensor for symbol_values in differentiator docstrings
**Python: 3.8
TFQ: 0.4.0**

I was trying to see what kind of speed up I would get from using `tfq.differentiators.Adjoint`. The [docstring example](https://www.tensorflow.org/quantum/api_docs/python/tfq/differentiators/Adjoint) runs fine, but when I add more parameters to the circuit, I get the following error message
```python
Traceback (most recent call last):
  File "/home/rooler/Documents/tfq/kagome/does_adjoint_work.py", line 27, in <module>
    grads = g.gradient(expectations, symbol_values_tensor)
  File "/home/rooler/anaconda3/envs/tfq/lib/python3.8/site-packages/tensorflow/python/eager/backprop.py", line 1067, in gradient
    flat_grad = imperative_grad.imperative_grad(
  File "/home/rooler/anaconda3/envs/tfq/lib/python3.8/site-packages/tensorflow/python/eager/imperative_grad.py", line 71, in imperative_grad
    return pywrap_tfe.TFE_Py_TapeGradient(
  File "/home/rooler/anaconda3/envs/tfq/lib/python3.8/site-packages/tensorflow/python/ops/custom_gradient.py", line 450, in actual_grad_fn
    raise ValueError(
ValueError: ('custom_gradient function expected to return', 5, 'gradients but returned', 4, 'instead.')
```
Code to reproduce:
```python
import cirq
import tensorflow_quantum as tfq
import tensorflow as tf
import sympy
import numpy as np

my_op = tfq.get_expectation_op()
adjoint_differentiator = tfq.differentiators.Adjoint()
# Get an expectation op, with this differentiator attached.
op = adjoint_differentiator.generate_differentiable_op(
    analytic_op=my_op
)
qubit = cirq.GridQubit(0, 0)
circ = cirq.Circuit(cirq.X(qubit) ** sympy.Symbol('alpha'))
circ.append(cirq.Y(qubit) ** sympy.Symbol('beta'))
circuit = tfq.convert_to_tensor([circ])

psums = tfq.convert_to_tensor([[cirq.Z(qubit)]])
symbol_values_array = np.array([[0.123, 0.3]], dtype=np.float32)

# Calculate tfq gradient.
symbol_values_tensor = tf.convert_to_tensor(symbol_values_array)
with tf.GradientTape() as g:
    g.watch(symbol_values_tensor)
    expectations = op(circuit, ['alpha', 'beta'], symbol_values_tensor, psums
)
grads = g.gradient(expectations, symbol_values_tensor)
print(grads)
```

I am having some trouble trying to debug this. Any suggestions?

### PR DESCRIPTION
**Pull request by MichaelBroughton**
Fixes #427

### CODE REVIEW

File: tensorflow_quantum/python/differentiators/adjoint.py [hunk 1]
```diff
@@ -48,14 +48,15 @@ class Adjoint(differentiator.Differentiator):
...
     ... ])
     >>> psums = tfq.convert_to_tensor([[cirq.Z(qubit)]])
-    >>> symbol_values_array = np.array([[0.123]], dtype=np.float32)
+    >>> symbol_values = np.array([[0.123]], dtype=np.float32)
     >>> # Calculate tfq gradient.
-    >>> symbol_values_tensor = tf.convert_to_tensor(symbol_values_array)
+    >>> symbol_values_t = tf.convert_to_tensor(symbol_values)
+    >>> symbol_names = tf.convert_to_tensor(['alpha'])
     >>> with tf.GradientTape() as g:
     ...     g.watch(symbol_values_tensor)
```
Comment (zaqqwerty) [inline tensorflow_quantum/python/differentiators/adjoint.py:56]: update to `symbol_values_t`

**Review by zaqqwerty — CHANGES_REQUESTED** [review]
small change

File: tensorflow_quantum/python/differentiators/adjoint.py (hunk 1 above)
Comment (MichaelBroughton) [reply tensorflow_quantum/python/differentiators/adjoint.py]: Good catch. Fixed

**Review by zaqqwerty — APPROVED** [review]
LGTM

**Pull request merged**